app/editor/src/features/content/form/PropertiesContentForm.tsx

```diff
@@ -1,137 +1,253 @@
+      setUserTags(result?.split(', '));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [values.summary]);
+
+  /** add tags to formik value object if tag exists - better implementation later*/
+  React.useEffect(() => {
+    setValidTags(
+      tags
+        .filter((t1: ITagModel) => userTags?.some((t2: string) => t2 === t1.id))
+        .map((x) => ({ ...x, createdOn: undefined })),
+    );
+    setFieldValue('tags', validTags);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [userTags]);
+
   return (
     <Col style={{ margin: '3%' }}>
       <Row>
         <Col>
           <Row>
             <FormikSelect
               className="md"
-              isDisabled
               value={series && series.find((s: any) => s.value === values.seriesId)}
-              onChange={(e: any) => setContent({ ...content, seriesId: e.value })}
-              options={series}
+              onChange={(e: any) => {
+                setFieldValue('seriesId', e.value);
+              }}
+              options={series ?? []}
+              required
               name="seriesId"
               label="Series"
             />
             <FormikText disabled className="md" name="otherSeries" label="Other Series" />
           </Row>
           <Row>
-            <FormikSelect className="md" isDisabled name="eod" label="EoD Category" options={[]} />
+            <FormikSelect
+              className="md"
+              name="categories"
+              label="EoD Category"
+              options={categoryTypes}
+              value={
+                values.categories.length > 0
+                  ? categoryTypes.find((c) => c.value === values.categories[0].id)
+                  : undefined
+              }
+              onChange={(e: any) => {
+                // only supports one at a time right now
+                let contentCategories = [];
+                contentCategories.push(categories.find((c) => c.id === e.value));
+                setFieldValue('categories', contentCategories);
+              }}
+            />
             <FormikSelect isDisabled className="md" name="score" label="Score" options={[]} />
           </Row>
           <Row style={{ position: 'relative' }}>
             <Col>
-              <SelectDate
+              <FormikDatePicker
+                required
+                disabled
                 className="md-lrg"
-                name="date"
+                name="publishedOn"
                 label="Date"
                 selectedDate={values.publishedOn ?? ''}
                 value={values.publishedOn}
-                disabled
                 onChange={(date: any) => {
-                  setContent({ ...content, publishedOn: date });
+                  setFieldValue('publishedOn', date);
                 }}
               />
             </Col>
-            <Col style={{ marginTop: '0.45em' }}>
+            <Col style={{ marginLeft: '1%' }}>
```

Comment:
Need to get in a habit of moving styling to the styled-component.

Reply:
Yeah there's a bunch I need to move. Will create a tech debt ticket for myself. Also ready to merge!